app/views/alaveteli_pro/info_request_batches/_authority_list.text.erb

```diff
@@ -0,0 +1,15 @@
+<%# This template should be identical to _authority_list.html.erb, except
+    that the public body names are marked as html_safe, so that they don't get
+    escaped by our translation helpers (because this a text view so they don't
+    need to be and we don't want html entities in it. %>
+<% if public_bodies.size > 0 %>
+<% example_body_names = public_bodies.take(2).map(&:name) %>
+<%= n_('{{count}} recipient, {{first_public_body}}',
+       '{{count}} recipients, including {{first_public_body}} and {{second_public_body}}',
+       public_bodies.size,
+       count: public_bodies.size,
+       first_public_body: raw(example_body_names.first),
```

Comment:
So I initially used `html_safe` but it doesn't work when a body name is nil (which it might be in the singular case here), so used raw() instead. My understanding was that they're essentially interchangeable, and indeed it does seem to be doing something to the string that's passed into the translation helper, because without it the spec fails. I've updated the comment in 700dd45 to make all that clearer.